### PR TITLE
Feature agregar foto a wyelab 161017953

### PR DESCRIPTION
**Pull request by jpducassou**

### CODE REVIEW

File: src/views/CreateWyelab.jsx [hunk 1]
```diff
@@ -172,96 +199,152 @@ class CreateWyelab extends Component {
...
+            <Row className="justify-content-center">
+              <Col sm={12} lg={6}>
+                <Card>
+                  <CardHeader>
+                    <h5 className="title">Create Wyelab</h5>
+                  </CardHeader>
+                  <CardBody>
+                    <form onSubmit={this.handleSubmit}>
+
+
```
Comment (rocespedes) [inline src/views/CreateWyelab.jsx]: nit: unnecessary extra lines

File: src/views/CreateWyelab.jsx [hunk 2]
```diff
@@ -172,96 +199,152 @@ class CreateWyelab extends Component {
...
+                                  onChange: this.handleImageSelection,
+                                  tag: CustomInput,
+                                },
+                              },
+                            ]}
+                          />
+                        </Col>
+                      </Row>
+
+
```
Comment (rocespedes) [inline src/views/CreateWyelab.jsx]: nit: unnecessary extra lines

File: src/views/CreateWyelab.jsx [hunk 3]
```diff
@@ -9,21 +9,44 @@ import {
...
+    fReader.onload = () => {
+      const imagen = document.getElementById('wyelab-image');
+      imagen.src = fReader.result;
+      this.setState(prevState => ({
+        fields: {
+          ...prevState.fields,
+          inputImage: fReader.result,
+        },
+      }));
+      this.setState({ blocking: false });
```
Comment (luchong12) [inline src/views/CreateWyelab.jsx]: Cant `blocking` state be set in the previous  `this.setState` statement?

File: src/views/CreateWyelab.jsx (hunk 1 above)
Comment (jpducassou) [reply src/views/CreateWyelab.jsx]: Done

File: src/views/CreateWyelab.jsx (hunk 2 above)
Comment (jpducassou) [reply src/views/CreateWyelab.jsx]: Done

File: src/views/CreateWyelab.jsx (hunk 3 above)
Comment (jpducassou) [reply src/views/CreateWyelab.jsx]: Sure! Done. Thanks

**Pull request merged**